content/en/logs/guide/lambda-logs-collection-troubleshooting-guide.md

```diff
@@ -51,79 +48,45 @@ For more information on these and other AWS Lambda metrics, see [Amazon Lambda M
 
 ### Manage your function triggers
 
-The Lambda function needs to have triggers set up in order for logs to be forwarded. If you see [in your Lambda function's monitoring tab](#check-the-lambda-function-monitoring-tab) that the function is never invoked, it may not have any triggers set up. There are two ways to set triggers for the Lambda function: manual and automatic.
-
-{{< tabs >}}
-{{% tab "Manual trigger" %}}
-You can see if there are [manual triggers][1] set up for your Lambda function by looking directly in the Lambda function's Configuration tab as in the screenshot below:
-
-{{< img src="logs/guide/manual-triggers-example.png" alt="Example of manual triggers location"  style="width:80%;" >}}
-
-**Note** If you have triggers on your Lambda function but it still isn't being invoked, there may be a conflict with another resource already subscribed to the same log source. When you add a manual trigger, an error message informs you if a resource is already subscribed to the log source:
-
-{{< img src="logs/guide/creating-trigger-error-example.png" alt="Example of error when creating trigger with subscription"  style="width:80%;" >}}
-
-See [Check for conflicting subscriptions](#check-for-conflicting-subscriptions) for more information on removing subscriptions.
-
-[1]: https://docs.datadoghq.com/integrations/amazon_web_services/?tab=allpermissions#manually-setup-triggers
-{{% /tab %}}
-{{% tab "Automatic trigger" %}}
-
-You can see if there are [automatic triggers][1] set up for your Lambda function with the following steps:
+The forwarder Lambda function needs to have triggers (CloudWatch Logs or S3) set up in order for logs to be forwarded. Follow the steps below to ensure the triggers are set up correctly.
```

Comment:
For logs to be forwarded, the forwarder Lambda function needs to have triggers (CloudWatch Logs or S3) set up. Follow the steps below to ensure the triggers are set up correctly.